src/takeover.c

```diff
@@ -36,6 +37,7 @@
 
 static PgSocket *old_bouncer = NULL;
 
+// Not supported in multithread mode 
```

Comment:
FYI I think that's totally fine. This feature is deprecated, and I think we should probably remove it soon.